var/spack/repos/builtin/packages/r-lubridate/package.py

```diff
@@ -14,12 +14,15 @@ class RLubridate(RPackage):
     a consistent and memorable syntax that makes working with dates easy and
     fun."""
 
-    homepage = "https://cran.r-project.org/web/packages/lubridate/index.html"
-    url      = "https://cran.r-project.org/src/contrib/lubridate_1.7.1.tar.gz"
-    list_url = "https://cran.r-project.org/src/contrib/Archive/lubridate"
+    homepage = "https://cloud.r-project.org/package=lubridate"
+    url      = "https://cloud.r-project.org/src/contrib/lubridate_1.7.1.tar.gz"
+    list_url = "https://cloud.r-project.org/src/contrib/Archive/lubridate"
 
+    version('1.7.4', sha256='510ca87bd91631c395655ee5029b291e948b33df09e56f6be5839f43e3104891')
+    version('1.7.3', sha256='2cffbf54afce1d068e65241fb876a77b10ee907d5a19d2ffa84d5ba8a2c3f3df')
     version('1.7.1', '17dcb4c6a95189941bbdcffecf61b83b')
     version('1.5.6', 'a5dc44817548ee219d26a10bae92e611')
 
-    depends_on('r-rcpp@0.11:', type=('build', 'run'))
+    depends_on('r@3.0.0:', type=('build', 'run'))
+    depends_on('r-rcpp@0.12.13:', type=('build', 'run'))
     depends_on('r-stringr', type=('build', 'run'))
```

Comment:
> SystemRequirements:	A system with zoneinfo data (e.g. /usr/share/zoneinfo) as well as a recent-enough C++11 compiler (such as g++-4.8 or later). On Windows the zoneinfo included with R is used.

I wonder if zoneinfo is something we can package

Reply:
It would seem that most systems would have zoneinfo data.

Reply:
The zoneinfo data can be gotten from [IANA — Time Zone Database](https://www.iana.org/time-zones), so it could be packaged. I think this is something that is better handled at the system level though, unless there are a significant numbers of systems that use Spack that would not have the zoneinfo data installed.